config_template.cfg

```diff
@@ -1352,6 +1352,17 @@ FFD_TOLERANCE= 1E-10
 %
 % Maximum number of iterations in the Free-Form Deformation point inversion
 FFD_ITERATIONS= 500
+
+% Parameters for prevention of self-intersections within FFD box
+FFD_INTPREV = YES
+FFD_INTPREV_ITER = 10
+FFD_INTPREV_DEPTH = 3
+
```

Comment:
Here in the config template I would prefer to have an explanation for each option. Imo the best way is to have a suitable explanation in the CConfig.cpp and just pasting it here.